app/App/Account/Account/Balances/Balance/index.js

```diff
@@ -50,30 +52,24 @@ class Balance extends React.Component {
           <div className='signerBalanceCurrency'>
             {name}
           </div>
-          <div className='signerBalanceValue' style={(balance.displayBalance || '0').length >= 12 ? { fontSize: '15px', top: '10px' } : {}}>
-            <span 
-              className='signerBalanceSymbol'
-            >
-              {symbol}
-            </span>
-            <span
-              style={(balance.displayBalance || '0').length >= 12 ? { marginTop: '-3px' } : {}}
-            >
-              {balance.displayBalance}
-            </span>
+          <div className='signerBalanceValue'>
+            <DisplayValue 
+              type='ether' 
+              value={balanceValue} 
+              valueDataParams={{ decimals }} 
+              currencySymbol={symbol}
+            />
           </div>
           <div className='signerBalancePrice'>
             <div className='signerBalanceOk'>
               <span className='signerBalanceCurrentPrice'>
-                {svg.usd(10)}{balance.price}
+                <DisplayValue type='fiat' value={`1e${decimals}`} valueDataParams={{ decimals, currencyRate, isTestnet, displayFullValue: true }} currencySymbol='$' />
```

Comment:
what is `value` in this context? we used to show the price, what are we showing now?

Reply:
What price did we used to show?  The value here is one unit of <x>, the other parameters being required to display that value in terms of fiat currency for a given currencyRate.  We could probably wrap this

Reply:
There you go, now it shows the price